modules/admin_manual/pages/_partials/maintenance/upgrading/upgrade_steps.adoc

```diff
@@ -0,0 +1,63 @@
+= Upgrade PHP to Version {to-version}
+:redhat-software-collections-overview-url: https://developers.redhat.com/products/softwarecollections/overview
+
+To upgrade to PHP {to-version} you first need to subscribe to {redhat-software-collections-overview-url}[the Red Hat Software Collections] channel repository to download and install the PHP {to-version} package in RHEL 7 (if you've not done this already). 
+This documentation uses the same command as you will find there.
+
+[IMPORTANT]
+====
+Ensure that you have `subscription-manager` installed.
+If you don't, yet, have it installed, do so with the following command:
+
+[source,console]
+----
+yum install --assumeyes subscription-manager
```

Comment:
It is still required to run `subscription-manager repos --enable rhel-server-rhscl-7-rpms` otherwise no packages can be retrieved from the Software Collections repo.

Reply:
Thanks @pako81. I'll get that added to the documentation.

Reply:
I keep running into "_System certificates corrupted. Please reregister._" on the CentOS Docker container that I'm using, mainly because it doesn't have NTP enabled and configured.